content/installation/java/package/maven-cargo.md

```diff
@@ -0,0 +1,193 @@
+
+```xml
+<contrast.version>3.6.2.7576</contrast.version>
+```
+
+You can replace the version and build numbers with those from any Contrast Java agent [released to Maven Central](https://search.maven.org/artifact/com.contrastsecurity/contrast-agent).
+
+* Then add the following to the `build.plugins` element of the project POM (`pom.xml`):
+
+```xml
+<plugin>
+  <artifactId>maven-dependency-plugin</artifactId>
+  <version>3.1.1</version>
+  <executions>
+    <execution>
+      <phase>prepare-package</phase>
+      <goals>
+        <goal>copy</goal>
+      </goals>
+      <configuration>
+        <artifactItems>
+          <artifactItem>
+            <groupId>com.contrastsecurity</groupId>
+            <artifactId>contrast-agent</artifactId>
+            <version>${contrast.version}</version>
+          </artifactItem>
+        </artifactItems>
+        <stripVersion>true</stripVersion>
+      </configuration>
+    </execution>
+  </executions>
+</plugin>
+```
+
+## Step 6: Enable the Contrast agent
+
+Configure the existing Maven Cargo Plugin to include Contrast when starting Jetty. 
+
+* Copy the following `configuration` element to the `start-app-before-IT` execution:
+
+```xml
+<configuration>
+  <configuration>
+    <properties>
+      <cargo.jvmargs>
+        -javaagent:${project.build.directory}/dependency/contrast-agent.jar
+        -Dcontrast.config.path=${user.home}/.contrast.yml
+        -Dcontrast.application.name=maven-cargo-how-to
+      </cargo.jvmargs>
+    </properties>
+  </configuration>
+</configuration>
+```
+
+* Change the value of `-Dcontrast.config.path` to match the path to the configuration file you created in **Step 2**.
+
+* Now re-run the tests. When the application container initializes, you'll see output indicating that Contrast has started.
+
+```
+$ ./mvnw clean verify
+[INFO] Scanning for projects...
+[INFO]
+[INFO] ---------< com.contrastsecurity.examples:contrast-maven-cargo >---------
+[INFO] Building Contrast Maven Cargo Example 1.0
+[INFO] --------------------------------[ war ]---------------------------------
+[INFO]
+    ... omitting some output ...
+[INFO] [talledLocalContainer] [Contrast] Thu Mar 21 23:20:40 EDT 2019 Starting Contrast (build 3.6.2.BACKGROUND) Pat. 8,458,789 B2
+    ... omitting more output...
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  16.644 s
+[INFO] Finished at: 2019-03-21T22:57:22-04:00
+[INFO] ------------------------------------------------------------------------
+```
+
+> **Note:** On Windows, run `mvnw.cmd clean verify` instead.
+
+During the integration test, the agent detects and reports the vulnerable servlet to the Contrast UI. To see the vulnerability report, go to the [**Vulnerabilities** grid](user-vulns.html#discover-vulns) in the Contrast UI, and filter your view by the application name `maven-cargo-how-to`.
```

Comment:
@gilday @rszewczyk Why would the application name always be the same in the Contrast UI?

Reply:
Because the name is explicitly provided to the agent via configuration in Step 6

```
-Dcontrast.application.name=maven-cargo-how-to
```